sdk/template/Azure.Template/tests/MiniSecretClientLiveTests.cs

```diff
@@ -8,18 +8,26 @@
 
 namespace Azure.Template.Tests
 {
+    [ClientTestFixture(
```

Comment:
Most services start with a single version. It's a simple incremental change to switch to multi-version testing.

Reply:
I know. I wanted to put this here again to emphasize how one would support this. The presence of a single version here doesn't hurt anything - all the files generate the same.

Reply:
As discussed offline, I'll mention it in a comment and link to the docs.